tests/spot/convert/test_convert_trade_history.py

```diff
@@ -0,0 +1,48 @@
+import responses
+from urllib.parse import urlencode
+from tests.util import random_str
+from tests.util import mock_http_response
+from tests.util import current_timestamp
+from binance.spot import Spot as Client
+from binance.error import ParameterRequiredError
+
+mock_item = {"key_1": "value_1", "key_2": "value_2"}
+key = random_str()
+secret = random_str()
+
+
+params = {
+    "startTime": current_timestamp(),
+    "endTime": current_timestamp(),
+}
+
+
+def test_convert_trade_history_without_startTime():
+    """Tests the API endpoint to retreive convert trade history without startTime"""
+
+    params = {"startTime": "", "endTime": "1597130241000"}
+    client = Client(key, secret)
+    client.convert_trade_history.when.called_with(**params).should.throw(
```

Comment:
we can directly use `called_with("", 1597130241000)` here

Reply:
I changed it to a function call with a keyword argument. I figure this was clearer, just according to pythons "explicit is better than implicit".